server/daos/books.js

```diff
@@ -2,16 +2,8 @@ import db from '@database/models';
 
 export const insertBook = args => db.books.create(args);
 
-export const updateBook = async args => {
-  const mapBookUpdateArgs = {
-    name: args.name,
-    genres: args.genres,
-    pages: args.pages
-  };
+export const updateBook = async ({ id, name, genres, pages, publisherId }) => {
+  await db.books.update({ name, genres, pages, publisherId }, { where: { id } });
```

Comment:
same question here, do we always need the updated value?

Reply:
yeah we need the updated value